evaluate.py

```diff
@@ -29,33 +30,38 @@ def main(cfg: DictConfig) -> None:
             class_names.append(class_name)
 
             print(f'\n--> Plotting density for class ({class_name})')
-            fig_density_name = f'density_{class_name}.pdf'
-            fig_density = plot_class_score(df_pred, class_id, cfg["class_to_info"], how='density')
-            fig_density.write_image(fig_density_name)
-            mlflow.log_figure(fig_density, f'plots/{cfg["dataset"]}/density_{class_name}.html')
-            mlflow.log_artifact(fig_density_name, f'plots/{cfg["dataset"]}/pdf')
-            os.remove(fig_density_name)
+            try:
```

Comment:
Since in some cases there were cases, for example for the low pT Taus, if there were only 1 or 0 entries for the density plot, then it doesn't work, and the program ends. Since it was run through a script, I had to add a pass statement after catching the error.

The error was mainly due to the KDE part of the density plot, which requires at least 2 entries.